react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { openedTopic: null };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    const openedTopic = this.props.topics.filter(topic => {
+      if (topic.header === event.target.innerText && topic !== this.state.openedTopic) {
+        return true;
+      } else {
+        return false;
+      }
+    });
+    if (openedTopic === []) {
```

Comment:
Two arrays will never equal each other. You can check the length for `0` though.